update 2-1-blog (PR from TinaCMS) (#2091)
tinacms/tina.io: content/blog/TinaCMS-Version-21.mdx

### What's New:

1. **API - Removed Deprecated Hook**

>    * A deprecated hook, which was part of legacy internal systems, has now been fully removed. This change does not affect the commonly used `useTina` hook.

Reviewer: feels odd that these are bullet points, when there is only a single item in each section - recommend removing them

Reviewer 2: Fixed - 
![image](https://github.com/user-attachments/assets/779dd8e2-8d51-413c-9638-4a38cb25b7ec)

2. **React Framework - Next.js Version Upgrade**
   * We've upgraded the Next.js version in TinaCMS packages from 12 to 14, along with several other dependency updates to ensure compatibility and improved performance.
3. **Package Manager - pnpm Support for `create-tina-app`**

>    * You can now use pnpm as an option when creating a Tina app, offering more flexibility and efficiency in your package management.

Reviewer: ```suggestion
   * You can now use pnpm as an option when creating a Tina app, offering more choices with package managers when starting a new project.
```

4. **React 18 Integration**
   TinaCMS has been upgraded to use React 18.3.1, bringing you the latest features and improvements from React.
5. **Markdown Editor - New `toolbarOverride` Option for Rich Text Fields**

>    A new `toolbarOverride` option has been added to rich text fields, giving you greater control over your editor toolbar configurations. Check out the [docs](/docs/reference/types/rich-text/) for more info.\

Reviewer: ```suggestion
   A new `toolbarOverride` option has been added to rich text fields, giving you greater control over your editor toolbar configurations. Check out the [docs](/docs/reference/types/rich-text/) for more info.
```

   ![](https://res.cloudinary.com/forestry-demo/image/upload/v1721881770/blog-media/2.1/ray-so-export_3_x7baul.png)\
   ![](https://res.cloudinary.com/forestry-demo/image/upload/v1721881650/blog-media/2.1/SCR-20240725-mvmu_gfqxxv.png)
6. **Media Manager - Adding Support for Videos & Other Field Types**
   TinaCMS now allows you to upload and use more file types, e.g. PDF, videos, etc.
   ![](https://res.cloudinary.com/forestry-demo/image/upload/v1721880641/blog-media/2.1/351543729-f4b352d6-83c9-4f13-af38-8faa0e41f993_q8kuz7.png)
7. **Mobile Resolution Support**

>    TinaCMS now supports various screen sizes and mobile devices, including iOS and Android, providing a seamless editing experience across all your devices.\

Reviewer: ```suggestion
   TinaCMS now supports various screen sizes and mobile devices, including iOS and Android, providing a seamless editing experience across all your devices.
```
